### PR TITLE
Linux Doc cosmetic fixes.

### PR DESCRIPTION
**Pull request by fujitatomoya**
minor doc update for Linux Users.

### CODE REVIEW

**Comment by fujitatomoya** [comment]
friendly ping.

File: docs/linux.md [hunk 1]
```diff
@@ -49,16 +50,18 @@ User=ollama
 Group=ollama
 Restart=always
 RestartSec=3
+Environment="PATH=$PATH"
```
Comment (jmorganca) [inline docs/linux.md:53]: Do we need this line?
```suggestion
```

File: docs/linux.md [hunk 2]
```diff
@@ -49,16 +50,18 @@ User=ollama
...
 WantedBy=default.target
 ```
 
-Then start the service:
+Then start and check the service:
 
 ```bash
 sudo systemctl daemon-reload
 sudo systemctl enable ollama
+sudo systemctl status ollama
```
Comment (jmorganca) [inline docs/linux.md:64]: I don't think this line is required. The "Start Ollama" section below should cover it.
```suggestion
```

File: docs/linux.md (hunk 1 above)
Comment (fujitatomoya) [reply docs/linux.md:53]: I think we do, see

https://github.com/ollama/ollama/blob/94fff5805fecd25287e1a21ac9910859011adcff/scripts/install.sh#L134

File: docs/linux.md [hunk 3]
```diff
@@ -78,10 +80,11 @@ Make sure to install ROCm v6
 
 ### Start Ollama
 
-Start Ollama using `systemd`:
+Start and Check Ollama service using `systemd`:
 
 ```bash
 sudo systemctl start ollama
+sudo systemctl status ollama
```
Comment (fujitatomoya) [inline docs/linux.md:87]: I believe that would be nice to check if the ollama system service is running active here. @jmorganca how about adding status check here?

File: docs/linux.md (hunk 2 above)
Comment (fujitatomoya) [reply docs/linux.md]: removed with https://github.com/ollama/ollama/pull/6430/commits/2abc330d96faffff6d11dc1a3128205753c5d8cc

**Comment by fujitatomoya** [comment]
@jmorganca thanks for the review! is there anything i should address on my side?

**Comment by jmorganca** [comment]
Small tweak otherwise LGTM

File: docs/linux.md [hunk 4]
```diff
@@ -78,10 +80,11 @@ Make sure to install ROCm v6
 
 ### Start Ollama
 
-Start Ollama using `systemd`:
+Start and Check Ollama service using `systemd`:
```
[the remark this answers is not in the document]
Comment (fujitatomoya) [reply docs/linux.md]: Done with https://github.com/ollama/ollama/pull/6430/commits/b3a737e1cd4e13de8395d786bf24fb98d1ed744f

**Comment by fujitatomoya** [comment]
@jmorganca all comments are resolved.

**Review by jmorganca — APPROVED** [review]
Thanks!

**Pull request merged**